src/components/menu-surface/menu-surface.tsx

```diff
@@ -30,6 +30,12 @@ export class MenuSurface {
     @Prop()
     public open = false;
 
+    /**
+     * use material design's elevation styling
+     */
+    @Prop()
+    public elevationDesign: boolean;
```

Comment:
I will remove this, I don't think we should expose material internals and we should be able to just set the elevation all the time. It's probably just an inconsistency that it was not set for the `select` component